src/vhip/mlmodel/gene_features.py

```diff
@@ -7,7 +7,9 @@
 """
 
 import os
-from typing import List
+from typing import List, Union
+
+import scipy  # type: ignore
```

Comment:
Please specify the ignore (i.e., # pyright: ignore[reportMissingTypeStubs])

Reply:
added!